src/puptoo/process/profile.py

```diff
@@ -44,6 +45,20 @@ def catch_error(parser, error):
     logger.error(log_msg, parser, error)
 
 
+# GCP_CONFIRMED_CODES are the available marketplace license codes available
```

Comment:
lol

Reply:
> 
> 
> Logic looks good, just have some questions first then happy to approve.
> 
> Also do you have a schema PR and an insights-host-inventory PR in the works to accompany this?
> 
> Thanks!

We actually don't need a new schema item since all this does is flag the "is_marketplace" sys profile item that alreayd exists